release_notes/topics/19_0_0.adoc

```diff
@@ -0,0 +1,27 @@
+= OpenID Connect and SAML Adapters End-of-life
+
+Some Keycloak OpenID Connect adapters have reached end-of-life and are not included in this release.
+
+== Fuse 6 and 7 (OpenID Connect)
+
+Keycloak will no longer be providing adapters for Fuse 6 or 7. If you need adapters for Fuse please leverage https://access.redhat.com/products/red-hat-single-sign-on[Red Hat Single Sign-On] 7.x adapters.
+
+== JBoss AS 7 and EAP 6 (OpenID Connect and SAML)
+
+JBoss AS 7 has been unmaintained for a very long time. If you are still using JBoss AS 7 we recommend migrating to WildFly and leveraging the native OIDC support in WildFly.
+
+Red Hat customers using Red Hat JBoss Enterprise Application Platform 6.x should use https://access.redhat.com/products/red-hat-single-sign-on[Red Hat Single Sign-On] 7.x adapters. These can be used in combination with the Keycloak server.
```

Comment:
Can this note (and similar below) be read in a way that we guarantee the compatibility of all future Keycloak server versions with the RHSSO 7.x OIDC adapter?

Reply:
Our recommended upgrading process is to upgrade the server first, then adapters, so we'll have to keep compatibility I'm afraid. In addition it can't be a requirement that folks upgrade all their applications, say for example someone has some legacy app on EAP 6.4 that they're not planning on migrating, they can't be blocked from upgrading Keycloak/RH-SSO.